package.json

```diff
@@ -59,7 +58,8 @@
     "to-markdown": "^1.2.1",
     "underscore": "^1.8.3",
     "uswds": "git+https://github.com/18F/web-design-standards.git#v0.9.x",
-    "validator": "^3.39.0"
+    "validator": "^3.39.0",
+    "yamljs": "^0.2.4"
```

Comment:
There was a problem with browserifying `js-yaml` for some reason. We experienced the same thing in 18F/about-yml-npm#3 so I just switched to the other library here as well.